src/components/User.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+const data = require('../data/data.json')
+
+const users = data
+
+User.propTypes = {
+  userData: PropTypes.arrayOf(
+    PropTypes.shape({
+      name: PropTypes.string.isRequired,
+      phone: PropTypes.string.isRequired,
+      mail: PropTypes.string,
+      department: PropTypes.string.isRequired,
+    })
+  ),
```

Comment:
This is not an array but single properties - at least that's how I understood what you give as props in `UsersPage`. 
```suggestion
      name: PropTypes.string.isRequired,
      phone: PropTypes.string.isRequired,
      mail: PropTypes.string,
      department: PropTypes.string.isRequired,
```